[2023-09-03T22:56:43.820+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: myFirstDag_v5.third_task manual__2023-09-03T22:56:40.912668+00:00 [queued]>
[2023-09-03T22:56:43.834+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: myFirstDag_v5.third_task manual__2023-09-03T22:56:40.912668+00:00 [queued]>
[2023-09-03T22:56:43.834+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-03T22:56:43.853+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2023-09-03 22:56:40.912668+00:00
[2023-09-03T22:56:43.860+0000] {standard_task_runner.py:57} INFO - Started process 962 to run task
[2023-09-03T22:56:43.864+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'myFirstDag_v5', 'third_task', 'manual__2023-09-03T22:56:40.912668+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpsi5zsbd4']
[2023-09-03T22:56:43.867+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask third_task
[2023-09-03T22:56:43.924+0000] {task_command.py:415} INFO - Running <TaskInstance: myFirstDag_v5.third_task manual__2023-09-03T22:56:40.912668+00:00 [running]> on host 01961fb83b4d
[2023-09-03T22:56:44.033+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mohghaff' AIRFLOW_CTX_DAG_ID='myFirstDag_v5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-03T22:56:40.912668+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-03T22:56:40.912668+00:00'
[2023-09-03T22:56:44.035+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-03T22:56:44.036+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo This is the third task which will run simultaneoudly as task 2 after task 1.']
[2023-09-03T22:56:44.048+0000] {subprocess.py:86} INFO - Output:
[2023-09-03T22:56:44.050+0000] {subprocess.py:93} INFO - This is the third task which will run simultaneoudly as task 2 after task 1.
[2023-09-03T22:56:44.050+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-03T22:56:44.081+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=myFirstDag_v5, task_id=third_task, execution_date=20230903T225640, start_date=20230903T225643, end_date=20230903T225644
[2023-09-03T22:56:44.117+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-03T22:56:44.139+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
